feature/add consent models and fix temporal type

cutoffthetop opened the pull request:
# Added
- add consent models for model v3
- add `doi` field to resource models

# Changed
- update LOINC pattern

# Fixed
- fix temporal entity schemas

erichesse reviewed (commented):
Looks good. Please update the changelog as well, with the `DoiStr` which is not mentioned in the PR description as of yet.

cutoffthetop commented:
> Please update the changelog as well

the v3 branch also doesn't have a changelog yet. that would probably cause a load of merge conflicts anyway. but i added a flashy note to the v3 PR and added the `DoiStr` to both PR's descriptions
